backend/backend/settings.py

```diff
@@ -0,0 +1,130 @@
+"""
+Django settings for backend project.
+
+Generated by 'django-admin startproject' using Django 3.1.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '&ua(#u%8#8cbs0qvr4zp#plly77+e@x%7ov$b6%rl2x-hy^p5-'
```

Comment:
This is supposed to be secret. Exposing it is not best practice.

Check out this article - https://help.pythonanywhere.com/pages/environment-variables-for-web-apps/

Reply:
There are **libraries** you can use to access _environment variables_ in Python. 

This is one
```https://github.com/theskumar/python-dotenv```